pkg/gotohelm/testdata/src/example/typing/numbers.go

```diff
@@ -29,6 +32,10 @@ func numbers() []any {
 		-0.01,
 		123.00,
 		// math.MaxInt64 * math.MaxInt64, // Believe it or not, this actually causes an overflow error.
+		getMyInt() == 10,
+		getMyInt() == myint(getMyFloat()),
```

Comment:
How 10 can equal 1.1? Am I missing something?

Reply:
This isn't intended to evaluate to true. It just to run a comparison operation on these types and ensure it doesn't result in a runtime error in helm/template world.